etc/testing/test_worker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euxo pipefail
+
+export TIMEOUT="${1}"
+PROM_PORT="$(
+  kubectl --namespace=monitoring get svc/prometheus -o json \
+    | jq -r .spec.ports[0].nodePort
+)"
```

Comment:
This doesn't work on my machine and I'm not sure what I would need to do to get it to work.

Reply:
Also, I commented this out and the tests ran just fine?

Reply:
This is for TestPrometheusStats: https://github.com/pachyderm/pachyderm/blob/master/src/server/worker/stats/stats_test.go#L93

Peter and I disabled that test before we decided to move this change into its own PR, because deploying Prometheus sounded annoying and TBH I don't even know if anyone uses our Prometheus integration anymore. To get it to work, you need to run `make launch-stats`, which is one of the dependencies of the `test-worker` make target that runs this test.

TBH I'm not sure what the best thing to do here is. If we want to support prometheus stats, then we need to re-add this code and fix the test. If we don't want to support prometheus stats, we should delete `src/server/worker/stats/` and everything that calls it. I'll leave it like this, raise a question in eng, and deal with it in a followup PR

Reply:
Update: per slack, hub uses Prometheus stats. I'll try to fix this test in the PR then. Maybe after these changes are made I'll ask to do another review pass?
- [ ] add a note to `test_worker.sh` clarifying that it should only be run via `make` so that Prometheus is also running.
- [ ] Fix & deflake `TestPrometheusStats`